frontend/src/components/advanced-auto-filter/advanced-auto-filter.tsx

```diff
@@ -64,6 +72,31 @@ const AdvancedAutoFilter: FC<AdvancedAutoFilterProps> = (props) => {
     });
   };
 
+  const [filterContainer, setRef] = useState<HTMLDivElement | null>(null);
+
+  useLayoutEffect(() => {
+    if (!filterContainer) return;
+
+    const headerHeight =
+      document.querySelector('#mainHeader')?.clientHeight || 0;
+
+    const searchTitleHeight = getElementHeight(
+      document.querySelector('#searchTitle'),
+    );
```

Comment:
Why in the first case you just refer to the `clientHeight` and in the second one you use the `getElementHeight`. Because semantically they do the same thing.
If you would like to convey that `getElementHeight` does smth specific - use more specific and explicit name

Reply:
Ok, I can change name -> getElementHeightWithMargins